hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +48,52 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-## Prepare the USB mass storage device
+You can now boot from an USB Mass Storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+.
+
+## Raspberry Pi 3B+
+
+The Raspberry Pi 3B+ supports USB Mass Storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
+
+This is verified using the following steps, this is the same (procedure)[https://www.raspberrypi.org/documentation/installation/installing-images/] as for SD cards though many alternative ways are functional:
+
+1. Download Raspbian Buster Lite
+
+2. Unzip the .zip file (tested with 7zip, but any other should be fine) to an .img file.
+
+3. Initialise the drive (tested with MBR scheme, no partition/clean disk).
+
+4. Flash the .img file to the external drive (tested using Win32DiskImager on Windows 10)
+
+5. Unmount and disconnect the drive.
+
+6. Connect the drive to the Raspberry Pi and power up the Pi (mind extra USB power usage from external drive).
+
+7. After five to ten seconds, the Raspberry Pi should begin booting and show the rainbow splash screen on an attached display. Make sure that you do not have an SD card inserted in the Pi, since if you do, it will boot from that first.
+
+## Raspberry Pi 4B
+
+Support for USB mass storage boot will be added to the Raspberry Pi 4B in a future software update.
+
+According to rpdom in a [forum post](https://www.raspberrypi.org/forums/viewtopic.php?t=243995#p1488036):
+
+The Pi4 uses a different boot loader to the earlier models. It is stored in eeprom on the board instead of part in the chip and part on the SD card. An update and instructions on how to apply it will be issued when the USB and network boot is ready.
+
+## Extra info
+
+https://www.raspberrypi.org/blog/pi-3-booting-part-i-usb-mass-storage-boot/
```

Comment:
Ironically, this blog post link was removed by an earlier PR!

Reply:
To be honest it was a very usefull resource for getting my setup to work and a dev-blog isn't just a random blogpost in my opinion.

However; I've merged as much of the (relevant) info of that blogpost in this article; so we could indeed remove the link. (And I did)